scripts/dataset_processing/tts/preprocess_text.py

```diff
@@ -94,8 +109,11 @@ def main():
 
     input_manifest_path = args.input_manifest
     output_manifest_path = args.output_manifest
+    text_key = args.text_key
+    normalized_text_key = args.normalized_text_key
     lower_case = args.lower_case
     num_workers = args.num_workers
+    batch_size = args.batch_size
```

Comment:
Could this be renamed to something like `joblib_batch_size`?

`batch_size` is too ubiquitous in deep learning and it instantly makes me worry about GPU memory etc etc when seen in scripts like this

Reply:
Sure, makes sense to me.